src/covid19sim/interventions.py

```diff
@@ -1,10 +1,12 @@
+from datetime import timedelta
 from orderedset import OrderedSet
 import numpy as np
 
-from covid19sim.config import RHO, GAMMA, MANUAL_TRACING_P_CONTACT,\
-    RISK_TRANSMISSION_PROBA, BIG_NUMBER, COLLECT_TRAINING_DATA, DEFAULT_DISTANCE
+from covid19sim.configs.constants import BIG_NUMBER
+from covid19sim.configs.config import RHO, GAMMA, MANUAL_TRACING_P_CONTACT,\
+    RISK_TRANSMISSION_PROBA, DEFAULT_DISTANCE
 from covid19sim.models.run import integrated_risk_pred
-
+from covid19sim.configs.exp_config import ExpConfig
```

Comment:
Does this automagically load the config files? I see that there is loading in `run.py`, but this seems to be importing just the class. I am pretty new to `yaml` ofc!